Topological sorting.md

```diff
@@ -3,4 +3,5 @@ categories: Graph theory
 ...
 
 ## Problems
-- [Reactivity Series](https://open.kattis.com/problems/reactivity)
\ No newline at end of file
+* [Reactivity Series](https://open.kattis.com/problems/reactivity)
+* [Build deps](https://open.kattis.com/problems/builddeps)
```

Comment:
The title should be "Build Dependencies". Also, this is a very nice introductory problem, so maybe it should go above Reactivity Series?

Reply:
I did not know you ordered them in terms of difficulty. Good to know.

Reply:
I try to order them in a way that is useful for someone new to the topic. So some kind of order based on difficulty, simplicity and usefulness.